mbed_app.json

```diff
@@ -467,6 +470,7 @@
             "target.bootloader_img"                     : "bootloader/mbed-bootloader-gr_peach-internal_flash-no_rot-v4.1.2.bin",
             "target.header_offset"                      : "0x10000",
             "target.app_offset"                         : "0x10400",
+            "target.restrict_size"                      : "0xEFC00",
```

Comment:
Renesas [GR-PEACH](https://os.mbed.com/platforms/Renesas-GR-PEACH/) (RZ_A1H) has 8MB flash memory, so 4MB per bank (not 1MB).
https://github.com/ARMmbed/mbed-os-example-pelion/blob/32354d030e7e4bb87bd2f83c9f83adc60ab23065/mbed_app.json#L477

`target.restrict_size` should be `0x3EFC00`

Reply:
Good catch, fixed